### PR TITLE
[Bug] ref bug #130801 - Edit Group Name To Full Name Group

### PR DESCRIPTION
**Pull request by Tuanlvt**
Tại màn hình tạo mới request cá nhân, trường Group, thông tin Group của user chỉ hiển thị 1 mình tên group hiện tại mà user đang trực thuộc.Trong trường hợp này, mong muốn hiển thị đầy đủ thông tin của cả Division/Section/Group.
link redmine https://dev.framgia.com/redmine/issues/130801
Effect: 
 + Screen confim OverTime, Request Leave, Request Off
 + Screen Profile
 + Screen member request detail

### CODE REVIEW

File: app/src/main/res/layout/activity_request_leave.xml [hunk 1]
```diff
@@ -94,63 +94,54 @@
...
-                                tools:ignore="ContentDescription"
-                                style="@style/StyleImageDropDown"
+                            <EditText
+                                android:ellipsize="end"
+                                android:enabled="@{viewModel.visibleGroup}"
+                                android:hint="@string/group"
+                                android:inputType="text"
+                                android:maxLines="1"
+                                android:onClick="@{viewModel::onPickGroup}"
+                                android:paddingEnd="@dimen/dp_24"
```
Comment (trainer-android-fr) [inline app/src/main/res/layout/activity_request_leave.xml]: WARNING: [RtlSymmetry] When you define `paddingEnd` you should probably also define `paddingStart` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.

File: app/src/main/res/layout/activity_request_leave.xml [hunk 2]
```diff
@@ -94,63 +94,54 @@
...
+                                android:paddingRight="@dimen/dp_24"
```
Comment (trainer-android-fr) [inline app/src/main/res/layout/activity_request_leave.xml]: WARNING: [RtlSymmetry] When you define `paddingRight` you should probably also define `paddingLeft` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.

File: app/src/main/res/layout/activity_request_overtime.xml [hunk 3]
```diff
@@ -97,61 +97,52 @@
...
-                            <ImageView
-                                app:srcCompat="@drawable/ic_arrow_drop_down"
-                                tools:ignore="ContentDescription"
-                                style="@style/StyleImageDropDown"
+                            <EditText
+                                android:hint="@string/group"
+                                android:inputType="text"
+                                android:maxLines="1"
+                                android:onClick="@{viewModel::onPickGroup}"
+                                android:paddingEnd="@dimen/dp_24"
```
Comment (trainer-android-fr) [inline app/src/main/res/layout/activity_request_overtime.xml]: WARNING: [RtlSymmetry] When you define `paddingEnd` you should probably also define `paddingStart` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.

File: app/src/main/res/layout/activity_request_overtime.xml [hunk 4]
```diff
@@ -97,61 +97,52 @@
...
+                                android:paddingRight="@dimen/dp_24"
```
Comment (trainer-android-fr) [inline app/src/main/res/layout/activity_request_overtime.xml]: WARNING: [RtlSymmetry] When you define `paddingRight` you should probably also define `paddingLeft` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.

File: app/src/main/res/layout/activity_request_overtime.xml (hunk 3 above)
Comment (Tuanlvt) [inline app/src/main/res/layout/activity_request_overtime.xml]: Added

File: app/src/main/res/layout/activity_request_leave.xml (hunk 1 above)
Comment (Tuanlvt) [inline app/src/main/res/layout/activity_request_leave.xml]: Fixed

**Pull request merged**